src/data/data_file_handler.py

```diff
@@ -13,27 +13,26 @@
 
 
 class DataFileHandler(DataHandler):
-    def __init__(self):
-        super().__init__()
-        self._dataframe = pd.DataFrame()
+    def __init__(self, ignore: Optional[List] = None):
+        super().__init__(ignore)
 
-    def add_dataframes_from_folders_as_mean(self, folders: List, points: int):
+    def concat_dataframes_from_folders_as_mean(self, folders: List, points: int):
         for folder in folders:
             folder_path = Path(folder)
             folder_df = pd.DataFrame()
             match file_utils.get_files_from_folder_path(folder_path):
                 case Ok(data_files_names):
-                    folder_df = self._add_files_as_mean(folder_df, data_files_names, folder_path,
-                                                        points)
+                    folder_df = self._concat_files_as_mean(folder_df, data_files_names, folder_path,
+                                                           points)
                     folder_df["class"] = f"{folder_path.parent.name}/{folder_path.name}"
                     self._dataframe = pd.concat([self._dataframe, folder_df])
                 case Err(error):
                     logger.error(error)
 
-    def _add_files_as_mean(self, target_df: pd.DataFrame, data_files_path: List,
-                           location: Path, points: int) -> pd.DataFrame:
+    def _concat_files_as_mean(self, target_df: pd.DataFrame, data_files_path: List,
+                              location: Path, points: int) -> pd.DataFrame:
```

Comment:
If, for some reason, there's no match, the function will return Option[pd.DataFrame], I'd update that just to the on the safe side.
That'd also happen if `data_files_path` is empty.

Reply:
Hmm I am not sure if I am following you. `target_df` is a dataframe that you provide, to which the data will be concatenated. If for some reason any data was concatenated, you still will get the original `target_df`. So there is no option to return a `None`. Also, you mean `Optional` right?

Reply:
Ah, right, got it. I hadn't properly understood the function, and you just made it clear. Thanks :)

P.D.: Yes, I meant `Optional`.